test/app.test.js

```diff
@@ -0,0 +1,65 @@
+import App from '../pages/app'
+
+describe('<App /> component', () => {
+  it('<App /> renders correctly', () => {
+    const component = shallow(<App />)
+    expect(component).toMatchSnapshot()
+  })
+
+  it('Initial values of state objects', () => {
+    const component = shallow(<App />)
+    const { computerChoice, computerCount, humanChoice, humanCount, totalGames, roundWinner, ties } = component.state()
+    expect(computerChoice).toEqual(null)
+    expect(humanChoice).toEqual(null)
+    expect(computerCount).toEqual(0)
+    expect(humanCount).toEqual(0)
+    expect(totalGames).toEqual(0)
+    expect(roundWinner).toEqual(null)
+    expect(ties).toEqual(0)
+  })
+
+  it('Rendering of state values in the DOM before any interaction with the App', () => {
+    const component = shallow(<App />)
+    const computerChoice = component.find('.comp-choice').text()
```

Comment:
Never use class names to find components, these may change at any time and then break your test suite.  Add another attribute like `data-behavior='computerChoice'` something along those lines.  Follow this pattern throughout the rest of the test suite,